apps/galaxy/src/services/utils/axios.ts

```diff
@@ -3,15 +3,36 @@ import axios from "axios";
 import { environment } from "../constants/environment";
 import { getToken } from "@src/utils/tokenService";
 
-const token = getToken() || "";
 const REQUEST_TIMEOUT = 2000;
 
+type config = {
+  url: string;
+};
+
+const urlWhiteList = ["/staff/signin", "/franchisee/signup", "/staff/verify"];
```

Comment:
Naming this `urlWhiteList` is kinda werid, it feels like we are white/black listing URIs here, which is not really the case.

I suggest we rename this to `nonrestrictedURIs` or something, but if you still feels like `urlWhiteList` is more appropriate then that's kool with me as well.